tests/test_eval.py

```diff
@@ -2,28 +2,46 @@
 
 import pytest
 from hydra.core.hydra_config import HydraConfig
-from omegaconf import open_dict
+from omegaconf import DictConfig, open_dict
 
 from src.eval import evaluate
 from src.train import train
+from tests.helpers.run_if import RunIf
+
+test_experiments = [
+    (
+        ["experiment=cls_modelnet_pointnet++"],
+        ["model=cls_pointnet++", "data=cls_modelnet2048", "ckpt_path=."],
+        1,
+    ),
+    (
+        ["experiment=seg_s3dis1x1_pointnet++"],
+        ["model=seg_pointnet++", "data=seg_s3dis1x1", "ckpt_path=."],
+        2,
```

Comment:
is there a rational to parameterizing the max epochs for testing?

Reply:
Yeah for example the seg model takes more than one epoch to start seeing an improvement in accuracy

Reply:
KK I've removed that